Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/sorear/smetamath-rs"
 clap = "2.5.2"
 filetime = "0.1.10"
 fnv = "1.0.2"
+regex = "0.1.71"
```

Comment:
Well, also remember that SMM3 is tailored for being fast, not slim. There are plenty of other verifiers that do slim better. So if regex crate is big because it's fast, that seems like a good compromise.